src/components/UploadCSVView/CSVViewTable/CSVViewTable.module.css

```diff
@@ -17,7 +17,7 @@
 .csv-table-container {
   display: flex;
   flex-direction: column;
-  height: 80vh;
+  height: 65vh;
```

Comment:
Could we ask design if it's possible to move the Add to Map button to the top of the page instead? The button is still cut off on my screen :(

Reply:
Yeah let's ask them